### PR TITLE
{bio}[foss/2022a] HTSplotter v0.15 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3127.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/19ac7f76a02f6338afd242b898ebe330 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/h/HTSplotter/HTSplotter-0.15-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }),
+    ('pypdf', '3.4.1', {
+        'checksums': ['8b0badc787062e8fb0d8393eb2a76903717fb24e5cc84e2b03cde5ac3881700f'],
+        # break import from Crypto, since pycrypto 2.6.1 is not actually compatible with Python 3.x
+        'preinstallopts': "sed -i 's/from Crypto/from Cryptoxxx/g' pypdf/_encryption.py && ",
+    }),
+    ('PyPDF2', '3.0.1', {
+        'checksums': ['a74408f69ba6271f71b9352ef4ed03dc53a31aa404d29b5d31f53bfecfee1440'],
+        # break import from Crypto, since pycrypto 2.6.1 is not actually compatible with Python 3.x
+        'preinstallopts': "sed -i 's/from Crypto/from Cryptoxxx/g' PyPDF2/_encryption.py && ",
```
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/h/HTSplotter/HTSplotter-0.15-foss-2022a.eb:36]: @boegel please check if this can be removed, as Micket suggests.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/b78aed37aaa3a02f35597e6aa5f857de for a full test report.

**Comment by boegel** [comment]
@verdurin You need a reinstall of `Python` to replace `pycrypto` with `pycryptodome`, cfr. #17412

**Comment by verdurin** [comment]
Yes, underway.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/00ff39b4e304a3e5b53d351b2e8da652 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**